gum/arch-arm64/gumarm64writer.h

```diff
@@ -7,6 +7,9 @@
 #ifndef __GUM_ARM64_WRITER_H__
 #define __GUM_ARM64_WRITER_H__
 
+#include "gumlibc.h"
+#include "gummemory.h"
```

Comment:
Why are these included in the header and not in the implementation only – any types needed for the header?